src/components/WorkerList/Header/index.tsx

```diff
@@ -0,0 +1,24 @@
+import * as S from './style';
+
+import { FilterButton } from '@/components';
+import type { GenerationType } from '@/types';
+
+interface Props {
+  selectedGeneration: GenerationType;
+  setIsShowFilterModal: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const WorkerListHeader: React.FC<Props> = ({
+  selectedGeneration,
+  setIsShowFilterModal,
+}) => (
+  <S.Header>
+    <S.Title>취업자 리스트 조회</S.Title>
+    <FilterButton
+      selectedGeneration={selectedGeneration}
+      onClick={() => setIsShowFilterModal(true)}
```

Comment:
```suggestion
      onClick={() => setIsShowFilterModal((isShow) => !isShow)}
```

이런식으로 필터 버튼을 다시 눌렀을 때 모달창이 사라지도록 토글 형식으로 변경하는건 어떨까요?

Reply:
넵 좋은 제안 감사합니다

Reply:
https://github.com/GSM-Networking/GSM-Networking-client/pull/3/commits/53c5751c7e4da29dd06cea9cf8f6fff1526b5099

리뷰 이전에 수정을 해둔 상태로 push가 안되어 있었네요!

확인 한번 부탁합니다